python/cudf/cudf/tests/test_datetime.py

```diff
@@ -1299,6 +1300,24 @@ def test_is_leap_year():
     assert_eq(expect2, got2)
 
 
+@pytest.mark.parametrize("dtype", DATETIME_TYPES)
+def test_days_in_months(dtype):
+    nrows = 10000
+
+    data = dataset_generator.rand_dataframe(
+        dtypes_meta=[
+            {"dtype": dtype, "null_frequency": 0.4, "cardinality": nrows}
+        ],
+        rows=nrows,
+        use_threads=False,
+    )
```

Comment:
```suggestion
    data = dataset_generator.rand_dataframe(
        dtypes_meta=[
            {"dtype": dtype, "null_frequency": 0.4, "cardinality": nrows}
        ],
        rows=nrows,
        use_threads=False,
        seed=23
    )
```

Might aswell add seed to have deterministic output.